javascript/javascript2/week1/shortest-word.js

```diff
@@ -0,0 +1,9 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function findShortest(danishWords) {
+  return danishWords.reduce(function (a, b) {
```

Comment:
Try to put meaningful naming for function parameters. See bellow
```suggestion
  return danishWords.reduce(function (currentSmallestWord, currentWord) {
```